### ISSUE TITLE
As a CIF developer, I want the routinely updated widget to call the proper backend mutation, so that I can use it on the Project Status field

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-cif#1392]
#### Description:

This is about linking #1380 and #1382 and having the widget call the proper mutation

#### Acceptance Criteria:

Given I am a CIF developer
When I transform a form field to be routinely updated
Then I have a widget that I can use
And that widget calls the proper backend mutation

Given I am a CIF administrator
When I am on the project overview page of a project
Then I can update the 'Project Status' field throught the routinely updated fields mechanism

#### Development Checklist:

- [ ] Make sure the frontend widget has all the configuration it needs to call the backend mutation defined in #1380
- [ ] Adjust the configuration as needed
- [ ] Implement that behaviour on the Project Status field of the Project Overview form
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by Dianadec** [comment]
Which CIF epic does this belong to? @LindsayMacfarlane

**Comment by LindsayMacfarlane** [comment]
I'm not sure @Dianadec - I don't believe there is an Epic that nicely captures this. As a widget, could it go into reusable components?

**Comment by pbastia** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1392?planning-poker) with Zenhub @BCerki @dleard @gurjmatharu @mikevespi @sam-warren @Sepehr-Sobhani

**Comment by pbastia** [comment]
This card should be done before any other routinely updated fields card

**Comment by LindsayMacfarlane** [comment]
Note: this ticket should be done before the other routinely updated tickets.

**Comment by LindsayMacfarlane** [comment]
Backlog refinement (Feb 22, 2022): There are still a lot of outstanding questions to support moving forward with amendments and revisions. Taking a pause to do a more in-depth review/whiteboarding session before continuing with the implementation of this EPIC.

**Comment by pbastia** [comment]
@LindsayMacfarlane  @Dianadec This is part of routinely updated fields and can be iceboxed/retired for now

**Comment by LindsayMacfarlane** [comment]
Removed from current Amendment & Revisions EPIC.
